project.yml

```
@@ -207,48 +184,13 @@ targets:
     dependencies:
       - sdk: SwiftUI.framework
       - sdk: WidgetKit.framework
-      - framework: Carthage/Build/iOS/abseil.framework
-        embed: false
-      - framework: Carthage/Build/iOS/BoringSSL-GRPC.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FIRAnalyticsConnector.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseAnalytics.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseCore.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseCoreDiagnostics.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseCrashlytics.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseFirestore.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseInstallations.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseRemoteConfig.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleAppMeasurement.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleDataTransport.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleToolboxForMac.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleUtilities.framework
-        embed: false
-      - framework: Carthage/Build/iOS/gRPC-C++.framework
-        embed: false
-      - framework: Carthage/Build/iOS/gRPC-Core.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GTMSessionFetcher.framework
-        embed: false
-      - framework: Carthage/Build/iOS/leveldb-library.framework
-        embed: false
-      - framework: Carthage/Build/iOS/nanopb.framework
-        embed: false
-      - framework: Carthage/Build/iOS/PromisesObjC.framework
-        embed: false
-      - framework: Carthage/Build/iOS/Protobuf.framework
-        embed: false
+      - package: FirebaseAnalytics
+      - package: FirebaseAppDistribution-Beta
+      - package: FirebaseCrashlytics
+      - package: FirebaseFirestore
+      - package: FirebaseInstallations
+      - package: FirebaseMessaging
+      - package: FirebaseRemoteConfig
     postBuildScripts:
       - script: /bin/sh ./Scripts/FirebaseCrashlytics/run
```

```suggestion
      - script: ${BUILD_DIR%Build/*}SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/run
```

FYIですが、CrashlyticsのScriptもSPM越しに実行ファイルにアクセスできるため、`./Scripts/FirebaseCrashlytics/run`ファイルを削除することができます。(runファイルのバージョンも一緒に追従できて嬉しいです。)

ただSPMのcheckoutしたRepositoryにアクセスするためには、DerivedData内のビルド時に生成される作業ディレクトリをたどる必要があるため、`${BUILD_DIR%Build/*}`という実行時に指定されるパスからたどる必要があり、若干変則的なScriptになります。